python.d/nvidia_smi.chart.py

```diff
@@ -0,0 +1,286 @@
+# -*- coding: utf-8 -*-
+# Description: nvidia-smi netdata python.d module
+# Author: Steven Noonan (tycho)
+
+import copy
+import subprocess
+import threading
+import xml.etree.ElementTree as et
+import time
+from bases.FrameworkServices.SimpleService import SimpleService
+from bases.collection import find_binary
+
+CHART_TEMPLATES = {
+    'pci_bandwidth': {
+        'options': [None, 'PCI express bandwidth utilization', 'KB/s',
+                    'nvidia_smi', 'nvidia_smi', 'area'],
```

Comment:
A lot of (actually, I think all of) the lines you reformatted  as two lines were fine as is.  While PEP 8 itself insists on a 79 character line length limit, we deviate from that and limit it to at most 120 characters instead (most people have big enough screens these days that 120 still fits perfectly fine).

Sorry about any confusion regarding this, we're still in the process of getting the rest of our existing code reformatted to be compliant with our coding standards, and we don't yet have thing set up to show nice notifications about whether the code meets the coding standards or not.

Reply:
Also, just to clarify, it's fine to merge this as it currently is (assuming of course that @l2isbad agrees), I just wanted to mention that this particular case of reformatting wasn't entirely necessary.

Reply:
Done. 👍  I left the small changes to code because I dont think theres any advantage to overly long nested and complex lines of code, but for array literals and strings let them breath!